python/GafferSceneUI/InteractiveRenderUI.py

```diff
@@ -36,6 +36,70 @@
 
 import Gaffer
 import GafferScene
+import GafferUI
+
+##########################################################################
+# UI for the state plug that allows setting the state through buttons
+##########################################################################
+
+from collections import namedtuple
+
+ButtonLabels = namedtuple( 'ButtonLabels', 'startButton pauseButton')
+
+class InteractiveRenderingStateWidget( GafferUI.PlugValueWidget ) :
+
+	labelMap = {
+		GafferScene.InteractiveRender.State.Stopped : ButtonLabels('Start', 'Pause'),
+		GafferScene.InteractiveRender.State.Running : ButtonLabels('Stop', 'Pause'),
+		GafferScene.InteractiveRender.State.Paused : ButtonLabels('Continue', 'Paused') }
+
+	def __init__( self, plug, *args, **kwargs) :
+
+		self.plug = plug
+
+		row = GafferUI.ListContainer( GafferUI.ListContainer.Orientation.Horizontal )
+		GafferUI.PlugValueWidget.__init__( self, row, plug )
+
+		with row :
+			self.startButton = GafferUI.Button()
+			self.pauseButton = GafferUI.Button()
+
+			self.startConnection = self.startButton.clickedSignal().connect( Gaffer.WeakMethod( self._startStopCallback ) )
```

Comment:
Gaffer has a convention of naming things with the following pattern :

- Signals : `clickedSignal()`
- Connections : `__clickedConnection()`
- Slots : `__clicked()`

This helps communicate the relationships between each of the things, and makes navigating the code a bit more predictable. So although it's wordy, I'd suggest `__startClickedConnection` and `__startClicked` (or something that reflects the dual start/stop nature of the button).